graylog2-server/src/main/java/org/graylog2/inputs/transports/netty/HttpHandler.java

```diff
@@ -34,10 +34,18 @@
 import io.netty.handler.codec.http.HttpVersion;
 
 public class HttpHandler extends SimpleChannelInboundHandler<HttpRequest> {
+    public static final String DEFAULT_GELF_PATH = "/gelf";
     private final boolean enableCors;
+    private final String path;
 
     public HttpHandler(boolean enableCors) {
         this.enableCors = enableCors;
+        this.path = DEFAULT_GELF_PATH;
```

Comment:
I would suggest calling the other constructor here instead:
```suggestion
        this(enableColors,DEFAULT_GELF_PATH);
```

Reply:
@roberto-graylog Thanks for the feedback. I ended up rolling back all changes in `HttpHandler` since the same `/gelf` endpoint is now used for both bulk and non-bulk messages.